scripts/data_contamination/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3
+
+RUN mkdir -p /script/src
+
+WORKDIR /script/src
+
+COPY . .
+
+RUN pip install -r ./requirements.txt
```

Comment:
use `requirements-freeze.txt` instead of `requirements.txt`

Reply:
Sounds good, I added requirements-freeze file and use that instead